runtimes/huggingface/mlserver_huggingface/runtime.py

```diff
@@ -57,6 +57,15 @@ def __init__(self, settings: ModelSettings):
                     ),
                 )
 
+        if settings.max_batch_size != self.hf_settings.batch_size:
+            raise InvalidTranformerInitialisation(
```

Comment:
Instead of an error, would it make sense to just log here a warning instead (i.e. with `logger.warning()`)?

Reply:
Thanks! Glad to be of any help.

About the comment, I think we can do it both ways, however, as it will work even if the batch sizes don't match a warning might be a better choice. I changed it in the next commit.

But, I think maybe in a future version or in another PR it might be a better design option to just read the hf_batch size into the MLServer max_batch_size unless there is an important reason for covering `max_batch_size != max_batch_size` cases. I cannot think of such situations but there might be some situations in which it makes sense to separate these two, can you think of any?